ui/src/app/cohort-search/constant.ts

```diff
@@ -19,3 +19,13 @@ export const DOMAIN_TYPES = [
     // { name: 'Vitals',       type: 'vitals' },
     // { name: 'Temporal',     type: 'temporal' }
 ];
+
+export const PM_UNITS = {
```

Comment:
Just wanted to check that these units will always be this? Just because I know Chris Fonnesbeck had to do some unit conversions at some point because things weren't being recorded consistently.

Reply:
We've been told they should be consistent, but I'm sure there's always a possibility the data is using something different. AFAIK there's no way to verify the units for the data.

Reply:
Sounds reasonable!